pm_nq/pm_nq.xml

```diff
@@ -0,0 +1,253 @@
+        <conditional name="task">
+            <param argument="task" type="select" display="radio" label="Task" help="Task to perform with pm-nq. Either generate a displaced structures, or average those structures.">
+                <option value="w">Generate</option>
+                <option value="r">Average</option>
+            </param>
+            <when value="w">
+                <conditional name="phonon">
+                    <param argument="phonon_source_type" type="select" display="radio" label="Phonon Type" help="The format of the phonon file.">
+                        <option value="castep">CASTEP</option>
+                        <option value="dftb+">DFTB+</option>
+                    </param>
+                    <when value="castep">
+                        <param name="phonon_source_castep" type="data" format="txt" label="Phonon File" help="Source file for phonon modes."/>
+                        <param name="castep_file" type="data" format="castep" label="CASTEP Log (.castep)" help="The CASTEP log produced when generating the Phonon File."/>
+                    </when>
+                    <when value="dftb+">
+                        <param name="phonon_source_dftb" type="data" format="binary" label="Phonon File" help="Source file for phonon modes."/>
+                        <param name="gen_file" type="data" format="txt" label="Gen File" help="GEN formatted geometry file for DFTB+."/>
+                    </when>
+                </conditional>
+                <param argument="script_file" type="data" format="txt" optional="true" label="Script File" help="For convenience, this script file is copied to all output directories (optional)."/>
+                <param argument="method" type="select" display="radio" label="Method" help="Method used to calculate thermal average">
+                    <option value="independent">Independent</option>
+                    <option value="montecarlo">Monte Carlo</option>
+                </param>
+                <param argument="mu_symbol" type="text" value="H:mu" label="Muon Symbol" help="Custom CASTEP species that identifies the muon."/>
+                <param argument="mu_index" type="integer" value="-1" label="Muon Index" help="If the Muon Symbol is not present in the cell, then the atom in the cell with this index will be treated as the muon instead. Not used if the Muon Symbol is present. Defaults to the last atom in the structure."/>
+                <param argument="grid_n" type="integer" value="20" min="1" label="Grid Points" help="Number of points to sample along each phonon mode or thermal line."/>
+                <param argument="sigma_n" type="float" value="3" min="0" label="Sigma Number" help="Number of sigmas to sample in the harmonic approximation."/>
+                <expand macro="k_points_grid"/>
```

Comment:
These parameters feel less significant than the ones below them, so suggest moving them down the order.

Overall I find the current order a bit confusing as it doesn't really 'flow' from one theme to the next. Perhaps some `<section>`s could help to break up all the options as well. But getting 'flow' right is a bit of trial and error.

Reply:
For this and the above comment, the order is the same as the PyMuonSuite schema file. But agree that if a re-order  with sections improves comprehension then it's worth doing.